generators/client/templates/vue/src/main/webapp/app/admin/user-management/user-management.vue.ejs

```diff
@@ -34,7 +36,11 @@
         </thead>
         <tbody v-if ="users">
           <tr v-for="user in users" :key="user.id" :id="user.login">
-            <td><router-link tag="a" :to="{name: '<%= jhiPrefixCapitalized %>UserView', params: {userId: user.login}}">{{user.id}}</router-link></td>
+            <td>
+              <router-link custom v-slot="{ href, navigate }" :to="{name: '<%= jhiPrefixCapitalized %>UserView', params: {userId: user.login}}">
```

Comment:
As far as I understand, an `a` tag is the default, so no need to `custom`ize it here.

Reply:
Ah yes its not required. Thanks for finding out.